Make bullet plugin work again

haudren opened the pull request:
Hello,

As the Bullet Plugin was broken, I took the liberty of fixing it (Since November according to  issue #19 ).

I had to remove the old Python code (createInstance) and modify the signature of some functions to match the bullet provided by Ubuntu 14.04. I also had to remove the NO_MODULE option in the CMakeLists which completely broke the build.

In addition, I had to turn off the double precision setting, but I suspect that this is because the package shipped by Ubuntu has it turned off, and thus did not include it in the commits.

All the best,
Hervé

bchretien commented on src/BulletPlugin/BulletCollisionDetector.cpp:
```diff
@@ -586,8 +586,8 @@ void BulletCollisionDetectorImpl::detectCollisions(boost::function<void(const Co
-    btCollisionObjectWrapper objectWrapper1(0, object1->getCollisionShape(), object1, object1->getWorldTransform(), -1, -1);
-    btCollisionObjectWrapper objectWrapper2(0, object2->getCollisionShape(), object2, object2->getWorldTransform(), -1, -1);
+    btCollisionObjectWrapper objectWrapper1(0, object1->getCollisionShape(), object1, object1->getWorldTransform());
+    btCollisionObjectWrapper objectWrapper2(0, object2->getCollisionShape(), object2, object2->getWorldTransform());
```
Note: this change is wrong for Bullet >= 2.82 (cf. https://github.com/bulletphysics/bullet3/commit/9eddd42347b4d766fabaaff9f72d69632cb82f48).

haudren commented:
I added a BT_VER_GT_282 to CmakeLists in order to correctly detect API version. Also removed the (hopefully) useless BT_DOUBLE_PRECISION setting as it should come from pkg-config.

bchretien commented:
:+1:  Works for me.

s-nakaoka commented:
Thank you.
